config/settings.yml

```diff
@@ -42,7 +42,6 @@ zip_storage: '/tmp' # override in #{RAILS_ENV}.yml
 replication:
   audit_should_backfill: false
 
-total_worker_count: 117 # for okcomputer endpoint
```

Comment:
resque cruft from before we transitioned to sidekiq (`git grep total_worker_count` shows no usage anymore)